airflow/providers_manager.py

```diff
@@ -36,8 +35,12 @@
 log = logging.getLogger(__name__)
 
 
-def _load_schema() -> Dict:
-    return json.loads(importlib_resources.read_text('airflow', 'provider.yaml.schema.json'))
+def _create_validator():
+    schema = json.loads(importlib_resources.read_text('airflow', 'provider.yaml.schema.json'))
+    cls = jsonschema.validators.validator_for(schema)
+    cls.check_schema(schema)
```

Comment:
This should probably be at test-time only, not run time.

Reply:
Removed. We validate this file in the pre-commit check, so we don't need any additional tests.